tests/unit/test_metrics.py

```diff
@@ -13,6 +13,58 @@
 pytestmark = pytest.mark.unit
 
 
+@pytest.mark.docker_required
+@pytest.mark.pytorch_deepspeech
+@pytest.mark.slow
+def test_entailment():
+    """
+    Slow due to 1 GB file download and multiple model predictions
+    """
+    print("Loading Entailment Model")
+    metric = metrics.Entailment()
+    metric_repeat = metrics.Entailment()
+    assert metric.model is metric_repeat.model
+
+    print("Loading Target Transcripts")
```

Comment:
were the `print()` statements temporary or intended to make it into develop?

Reply:
Removed.